doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -113,12 +134,13 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.lcm.gen.01`	| General | The Architecture **shall** support zero downtime expansion/change of physical capacity (compute hosts, storage increase/replacement) |
-| `req.lcm.gen.02` | General	| The Architecture **shall** support plug-ins to manage deployment of workload to multiple sites and various technology stack. Multi-cloud support through hybrid, migrations, etc. |
-| `req.lcm.gen.03` | General |	The Architecture **shall** provide OpenStack Ironic Bare Metal provision to support VIM Instance provisioning (bootstrap), Management System, KVM host provisioning and Kubernetes host provisioning |
-| `req.lcm.adp.01` | Automated deployment | The Architecture **shall** allow the “cookie cutter” automated deployment to multiple sites. |
-| `req.lcm.adp.02` | Automated deployment | The Architecture **must** enable automated and incremental upgrade of the various software components in a manner that still allows tenant workloads to continue to operate. |
-| `req.lcm.cid.02` | CI/CD | The Architecture **shall** support integrating with CI/CD Toolchain for NFVI and VIM components Automation |
+| `req.lcm.gen.01`	| General | The Architecture **should** support zero downtime expansion/change of physical capacity (compute hosts, storage increase/replacement) |
+| `req.lcm.adp.01` | Automated deployment | The Architecture **should** allow the “cookie cutter” automated deployment to multiple sites. |
+| `req.lcm.adp.02` | Automated deployment | The Architecture **must** support hitless upgrades of software provided by the cloud provider so that running workloads are not impacted. |
```

Comment:
How is "hitless" defined? What about the cases where a reboot of a compute host is inevitable?

Reply:
You can upgrade using redundancy!

Reply:
But this redundancy has to be on workload layer in order to "survive" a reboot (or failure) of a compute host without **service** impact. But for sure there is an impact on the actual workload on the compute host. "... so that running workloads are not impacted" can not be ensured (given that we don't want to introduce VM migration as a must)

Reply:
One way to solve this is to establish a communication between the workload and the infrastruture and manage an upgrade workflow where the workload is live or cold migrated only when it is ready for it. [OpenStack Fenix project](https://wiki.openstack.org/wiki/Fenix) implements this.

Reply:
I will assume there is no Edit suggests for this one. I propose to keep origin text and incase more fixed needed, an issue can be added and a PR can be created to resolve it. (unless there is a conclusion further down the comments :)